Pipfile

```diff
@@ -39,7 +39,7 @@ openapi-spec-validator = "==0.2.8"
 pexpect = "==4.8.0"
 psutil = "==5.7.0"
 pydocstyle = "==3.0.0"
-pydoc-markdown = "==3.3.0"
```

Comment:
Rather remove it then. LEt's add a step in the `generate_api_docs` script which installs the version `0.3.3` from pypi. This version you lowered it to is broken!

Reply:
ok